build.sbt

```
@@ -1833,8 +1653,238 @@ lazy val runtime = (project in file("engine/runtime"))
   .dependsOn(`runtime-compiler`)
   .dependsOn(`connected-lock-manager`)
   .dependsOn(testkit % Test)
-  .dependsOn(`logging-service-logback` % "test->test")
-  .dependsOn(`runtime-test-instruments` % "test->compile")
+
+/** A project holding all the runtime integration tests. These tests require, among other things,
+  * the `org.enso.runtime` JPMS module, so it is easier to keep them in a separate project.
+  * For standard unit tests, use `runtime/Test`.
```

shouldn't this be lowercase?
```suggestion
  * For standard unit tests, use `runtime/test`.
```

Note that running uppercase results in:
```
[IJ]runtime/Test
[error] Expected ':'
[error] Expected '/'
[error] Not a valid key: Test (similar: test, testOnly, state)
[error] runtime/Test
```

---

`runtime/Test` refers to the Test *configuration*, not to a command. This configuration is located, under the directory `engine/runtime/src/test`